.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - commit-message:
+      prefix: "Changelog:All"
+    directory: /tests/acceptance/
+    package-ecosystem: pip
+    schedule:
+      interval: weekly
+  - commit-message:
+      prefix: "Changelog:All"
+    directory: /meta-mender-qemu/docker/
```

Comment:
@mzedel I am unsure about this one, the Dockerfile(s) are at `/meta-mender-qemu/docker/*/Dockerfile`

Reply:
I don't know it for a fact, but I'm pretty sure these would have to be added as individual entries in the dependabot file (so one with _qemux86-64_ and one with _vexpress-qemu_ as the directory)